assets/src/blocks/Splittwocolumns/SplittwocolumnsInPlaceEdit.js

```diff
@@ -0,0 +1,126 @@
+import { Fragment } from '@wordpress/element';
+import { RichText } from '@wordpress/block-editor';
+import { debounce } from 'lodash';
+
+const { __ } = wp.i18n;
+
+/**
+ * WYSIWYG in-place editor 
+ */
+export const SplittwocolumnsInPlaceEdit = ({attributes, charLimit, setAttributes}) => {
+  const {
+    title,
+    issue_description,
+    issue_link_path,
+    issue_link_text,
+    issue_image_src,
+    issue_image_title,
+    focus_issue_image,
+    tag_description,
+    button_text,
+    tag_name,
+    tag_link,
+    tag_image_src,
+    tag_image_title,
+    focus_tag_image
+  } = attributes;
+
+  const toAttribute = attributeName => value => setAttributes({
+    [attributeName]: value
+  });
+  const debounceToAttribute = attributeName => debounce(
+    toAttribute(attributeName), 400
+  );
+
+  return (
+    <section className="block-wide split-two-column">
+      <div className="split-two-column-item item--left">
+        {issue_image_src &&
+          <div className="split-two-column-item-image">
+            <img
+              src={issue_image_src}
+              alt={issue_image_title || ''}
+              style={{objectPosition: focus_issue_image}}
+            />
+          </div>
+        }
+        <div className="split-two-column-item-content">
+          <RichText
```

Comment:
In order to be consistent with the other WYSIWYG blocks (for example [ArticlesEditor](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/master/assets/src/blocks/Articles/ArticlesEditor.js#L115)), we should use `keepPlaceholderOnFocus={true}` in these RichText components (it also avoids weird behaviour on click)